faq.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>FAQ | 판다마켓</title>
+<!-- pretendard web-font: https://github.com/orioncactus/pretendard -->
+<link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css">
```

Comment:
```suggestion
<link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" media="print" onload="this.media='all'">

```
웹 폰트를 사용할 때, 성능에 대해서도 고민을 하면 좋습니다. 😸

1. media="print" 
브라우저가 이 스타일시트를 즉시 로드하지 않습니다. 대신, 페이지의 주요 콘텐츠가 먼저 로드됩니다. 즉, 필요에 따라 지연로딩을 하게 설정할 수 있어요.
2. onload="this.media='all' 
스타일시트가 완전히 로드된 후에 이 JavaScript 코드가 실행됩니다. 

이렇게 하는 경우, 초기 페이지 로드 시간 단축할 수 있습니다. (즉, 폰트 스타일시트가 페이지 렌더링을 차단하지 않습니다.)